pkg/zdns/lookup.go

```diff
@@ -68,7 +69,11 @@ func parseMetadataInputLine(line string) (string, string) {
 }
 
 func parseNormalInputLine(line string) (string, string) {
-	s := strings.SplitN(line, ",", 2)
+	r := csv.NewReader(strings.NewReader(line))
```

Comment:
Is this cheap enough that we're not worried about creating a new reader for every line? Docs look like expected to be reused across a whole file.